tests/test_xml_parser_main_metadata_together.c

```diff
@@ -120,6 +120,28 @@ test_cr_xml_package_iterator_00(void)
     g_assert_cmpint(parsed, ==, 2);
 }
 
+static void
+test_cr_xml_package_iterator_filelists_ext_00(void)
+{
+    int parsed = 0;
+    GError *tmp_err = NULL;
+    cr_Package *package = NULL;
+
+    cr_PkgIterator *pkg_iterator = cr_PkgIterator_new(
+        TEST_REPO_04_PRIMARY, TEST_REPO_04_FILELISTS_EXT, TEST_REPO_04_OTHER, NULL, NULL, NULL, NULL, &tmp_err);
```

Comment:
My understanding of this API then, is that if you want the full metadata, you substitute the path to `filelists_ext` instead of `filelists`?

Reply:
For the parser, this is the idea.